### PR TITLE
Do editorial review on configure KoP doc

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -25,69 +27,75 @@ Logger:
...
-| saslAllowedMechanisms                | A set of supported SASL mechanisms exposed by the broker     | PLAIN,<br>OAUTHBEARER |         |
-| kopOauth2AuthenticateCallbackHandler | The fully qualified name of a SASL server callback handler class that implements the <br>AuthenticateCallbackHandler interface, which is used for OAuth2 authentication. <br>If it's not set, the class will be Kafka's default server callback handler for <br>OAUTHBEARER mechanism: OAuthBearerUnsecuredValidatorCallbackHandler. |                       |         |
+| saslAllowedMechanisms                | A set of supported SASL mechanisms exposed by the broker.     | PLAIN,<br>OAUTHBEARER |         |
+| kopOauth2AuthenticateCallbackHandler | The fully qualified name of an SASL server callback handler class that implements the <br>AuthenticateCallbackHandler interface, which is used for OAuth2 authentication. <br>If it is not set, the class will be Kafka's default server callback handler for <br>OAUTHBEARER mechanism: OAuthBearerUnsecuredValidatorCallbackHandler. |                       |         |
```
Comment (BewareMyPower) [inline docs/configuration.md:98]: Why is `an` SASL server?

File: docs/configuration.md [hunk 2]
```diff
@@ -25,69 +27,75 @@ Logger:
...
+> **NOTE**
+> 
+> These limits are based on each connection.
 
 ## Prometheus
 
 | Name                                     | Description                                                  | Default |
 | ---------------------------------------- | ------------------------------------------------------------ | ------- |
-| kopPrometheusStatsLatencyRolloverSeconds | Kop metrics expose to prometheus rollover latency in seconds. | 60      |
+| kopPrometheusStatsLatencyRolloverSeconds | Kop metrics exposed toPprometheus rollover latency in seconds. | 60      |
```
Comment (BewareMyPower) [inline docs/configuration.md:63]:
```suggestion
| kopPrometheusStatsLatencyRolloverSeconds | Kop metrics exposed to Prometheus rollover latency in seconds. | 60      |
```

Did you just only want to change `expose` to `exposed`? I think `prometheus` or `Prometheus` is both okay.

**Review by BewareMyPower — CHANGES_REQUESTED** [review]

File: docs/configuration.md (hunk 1 above)
Comment (Huanli-Meng) [reply docs/configuration.md]: updated to a SASL, PTAL

File: docs/configuration.md (hunk 2 above)
Comment (Huanli-Meng) [reply docs/configuration.md]: Updated to exposed to prometheus rollover....PTAL

**Review by BewareMyPower — APPROVED** [review]

**Pull request merged**